client/packages/requisitions/src/ResponseRequisition/DetailView/ReponseStats/ResponseStoreStats.tsx

```diff
@@ -102,12 +50,12 @@ export const ResponseStoreStats: React.FC<ResponseStoreStatsProps> = ({
             alignItems="flex-start"
             width={predictedStockPercent}
           >
-            <Divider />
             <ValueBar
               value={stockOnHand}
               total={predictedStockLevels}
               label={t('label.stock-on-hand')}
               colour="gray.dark"
+              startDivider
```

Comment:
should have the `endDivider` too - it was tucked away in the component 🙄 
which means that the other two instances below also need the end divider.

Reply:
actually - the behaviour differs: if the first ValueBar has no value, it isn't rendered, which means that the first divider isn't shown. In the original version, the divider is shown regardless. I suggest updating the ValueBar, so that the start divider is shown regardless of whether the rest of the component is shown or not.

Reply:
My apologies, I missed this one.  Will rectify.

Reply:
oops 🙄  just realised that `endDivider` defaults to true! Ignore previous comment..

Reply:
No worries.  It did make me think about it a bit more though.